src/Navbar.jsx

```diff
@@ -30,7 +30,7 @@ const Navbar = ({ token, deleteToken }) => {
   const history = useHistory();
   const logout = async () => {
     try {
-      await logoutUser(token);
+      await logoutUser(token.token);
```

Comment:
should it be token.token? if so I think you need to rename some properties/variable

Reply:
agreed, should it not be token.uuid?

Reply:
Yeah, when we wrote the object in the backend it was written with naming like this:
`token = {userId: 1, token: "insert UUID String here"}`
so that is something we'd need the backend to change.

Reply:
In the meantime you can still rename the object, the backend doesn't assign that as its passed as a prop!

Reply:
> In the meantime you can still rename the object, the backend doesn't assign that as its passed as a prop!

So rename it in `api` for now and then in `navBar` once they change it?

Reply:
currently in app you are passing the prop as `token` you can rename this there and subsequently rename the prop which `navBar` recieves this way it wont be `token.token`. Or similarly when you pass the setToken down to Login you could set the token and userId in state seperately. therefore for your `styledLink` you won't have to pass token.userId